src/lib/relay/config.tsx

```diff
@@ -12,6 +12,8 @@ if (Emission && Emission.gravityAPIHost && Emission.metaphysicsAPIHost) {
   gravityURL = "https://api.artsy.net"
 }
 
+metaphysicsURL = "http://localhost:5001"
```

Comment:
Oh yea, once https://github.com/artsy/metaphysics/pull/853 is merged can undo this.

Reply:
You can change this setting from the dev menu nowadays!

Reply:
oooo!